aries-site/src/pages/templates/inline-notifications.mdx

```diff
@@ -0,0 +1,143 @@
+  when communicating information which might result in loss of functionality or
+  requires the user's attention and/or action. Examples might include a system
+  outage, critical error, or expiring subscription or license. [Global banner notifications](/templates/global-banner-notifications)
+  are persistent and require user action to acknowledge or dismiss.
+- **Immediate feedback** &ndash; When providing immediate feedback to a user's
+  action, a variety of tools are likely more appropriate.
+
+  - When a user is awaiting a response after clicking an action button, use the
+    [busy button]() which provides in-progress and success animations. If the
+    response time exceeds expectations or the API call fails, then an inline
+    notification is appropriate for providing status updates.
+  - When validating form input client-side, use [field-level form validation]()
+    when possible. If validation results in a form-level error, using an inline
+    notification is appropriate.
+
+## Anatomy
+
+An inline notification consists of:
+
+| Label | Region                       | Purpose                                                      |                  Required                   | Notes                                                                                                                  |
+| ----- | ---------------------------- | ------------------------------------------------------------ | :-----------------------------------------: | ---------------------------------------------------------------------------------------------------------------------- |
+| **1** | **Icon or status indicator** | Indicates the type of notification                           | <Checkmark a11yTitle="true" size="small" /> | <span>Normal, Warning, Critical, and Unknown [status indicators](/templates/status-indicator) can be displayed.</span> |
+| **2** | **Title**                    | Short, descriptive label identifying the notification        |                                             | Titles should only be used for inline notifications presenting page level summaries.                                   |
+| **3** | **Message**                  | Briefly describes the action, status, or additional context  | <Checkmark a11yTitle="true" size="small" /> |                                                                                                                        |
+| **4** | **Anchor**                   | Used to direct users to additional detail or related content |                                             |                                                                                                                        |
+| **5** | **Close button**             | Dismisses the notification                                   |                                             |                                                                                                                        |
```

Comment:
Should we add a note here that Close button should not be used in "Critical" notifications?

Reply:
Done.